modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2650,7 +2678,7 @@ def fillna(df):
                     }
                     return df.fillna(value=func_dict, **kwargs)
 
-                if self._modin_frame.has_materialized_dtypes:
+                if self.has_materialized_dtypes():
                     dtypes = self._modin_frame.dtypes
```

Comment:
```suggestion
                    dtypes = self.dtypes
```

Reply:
At the query compiler level, the types of `_modin_frame` are simply accessed:
```python
    @property
    def dtypes(self) -> pandas.Series:
        return self._modin_frame.dtypes
```

Reply:
I was looking at the BaseQueryCompiler class [dtypes](https://github.com/arunjose696/modin/blob/arun-sqc/modin/core/storage_formats/base/query_compiler.py#L1966) which has  to_pandas call as other query compilers have self._modin_frame.dtypes, I think this should be fine.